_posts/2013-01-14-aria-landmark-roles.md

```diff
@@ -37,48 +80,71 @@ You see the warning ([consider fixing/changing](https://www.paciellogroup.com/bl
-        </td>
+      <td>contentinfo</td>
+      <td>
+        <code>&lt;footer&gt;</code>
+        <br>(if not a child of a sectioning element or the <code>main</code>.)
+      </td>
     </tr>
     <tr>
-        <td>navigation</td>
-        <td>
-          &lt;nav&gt;
-        </td>
+      <td>form</td>
+      <td>
+        <code>&lt;form&gt;</code>
+        <br>(if provided an accessible name via <code>aria-label</code> or <code>aria-labelledby</code>)
+      </td>
     </tr>
     <tr>
-        <td>contentinfo</td>
-        <td>
-          &lt;footer&gt;
-        </td>
+      <td>main</td>
+      <td>
+        <code>&lt;main&gt;</code>
+        <br>(only use one per page)
+      </td>
     </tr>
     <tr>
-        <td>article</td>
-        <td>
-          &lt;article&gt;
-        </td>
+      <td>navigation</td>
+      <td>
+        <code>&lt;nav&gt;</code>
+      </td>
     </tr>
     <tr>
-        <td>complementary</td>
-        <td>
-          &lt;aside&gt;
-        </td>
+      <td>region</td>
+      <td>
+        <code>&lt;section&gt;</code>
+        <br> (if provided an accessible name via <code>aria-label</code> or <code>aria-labelledby</code>)
+      </td>
     </tr>
     <tr>
-        <td>region</td>
-        <td>
-          &lt;section&gt;
-        </td>
+      <td>search</td>
+      <td>
+        none
+      </td>
     </tr>
   </tbody>
 </table>
 
-The good news is most modern [desktop browsers](http://stevefaulkner.github.io/html-mapping-tests/) (except IE) support this mapping. However, [iOS Safari](https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks-example.html) does not. OUCH! For now, it’s probably best to implement the landmark roles and ignore these warnings.
\ No newline at end of file
+The majority of [modern browsers](http://www.html5accessibility.com/) (except <abbr title="Internet Explorer">IE</abbr>) support these mappings. But it's always beneficial to run your own tests to ensure the appropriate landmark role is being appropriately exposed to assistive technology.
+
+For example, as of July 2018, Safari and VoiceOver on macOS High Sierra do not properly expose the <code>contentinfo</code> role from a <code>footer</code> element. When presented with situations like this, ignoring conformance warnings, and adding a redundant role to an element, may be preferred to not exposing the correct landmark information.
+
+
+## Additional references
+For more information about landmarks and other concepts mentioned in this article, you can check out the following resources:
+
+* [Accessible Landmarks](https://www.scottohara.me/blog/2018/03/03/landmarks.html)
+* [Where to put your search role](http://adrianroselli.com/2015/08/where-to-put-your-search-role.html)
+* [HTML Conformance checking](https://developer.paciellogroup.com/blog/2015/08/short-note-on-html-conformance-checking/)
+* [W3C: Using ARIA landmarks to identify regions of a page](https://www.w3.org/TR/WCAG20-TECHS/ARIA11.html)
```

Comment:
Does this exist under WCAG 2.1? Probably doesn't matter but just curious.

Reply:
which are you referring to here?